Source/implementations/f7/Meadow.F7/Devices/F7CellNetworkAdapter.cs

```diff
@@ -157,6 +170,9 @@ protected void InvokeEvent(CellFunction eventId, StatusCodes statusCode, byte[]
 
                 Resolver.Log.Trace($"Cell connection error: {GetCellConnectionError()}");
                 break;
+            case CellFunction.NtpUpdateEvent:
+                RaiseNtpTimeChangedEvent();
```

Comment:
Has the OS already changed the current time here (so if I call DateTime.Now in an event handler, will the time be correct)?

Reply:
There's an open [PR](https://github.com/WildernessLabs/Meadow/pull/636) in the Meadow.OS that aims to support NTP on cell/ethernet. 

So, I think we can wait for the OS PR to be merged before merging this PR. But, let me know if you want any changes.